Makefile

```diff
@@ -204,6 +205,7 @@ build-linux: tools
 .PHONY: build-linux
 
 build-docker-localnode:
+	@build-linux
```

Comment:
This shouldn't call `build-linux`. The `localnode` image is a bit weird in that it doesn't actually contain a binary, it is mounted into the container via a volume. The `localnet-start` target should depend on `build-linux`, not this, given the current state of things.

Reply:
I do agree it is a bit odd. I would propose we build the binary within the docker image instead of building externally. Do you have a prefernce?

Reply:
Personally, mounting the binary works better for my local workflow. I can try out changes in local testnets much faster since I don't have to rebuild the image, I simply recompile the binary and restart the containers.

Reply:
made the change as suggested.